internal/flowcontrol/interface.go

```diff
@@ -24,5 +24,6 @@ type FlowControlManager interface {
 	AddBytesSent(streamID protocol.StreamID, n protocol.ByteCount) error
 	SendWindowSize(streamID protocol.StreamID) (protocol.ByteCount, error)
 	RemainingConnectionWindowSize() protocol.ByteCount
-	UpdateWindow(streamID protocol.StreamID, offset protocol.ByteCount) (bool, error)
+	UpdateStreamWindow(streamID protocol.StreamID, offset protocol.ByteCount) (bool, error)
```

Comment:
Do we actually still need the bool return value? Seems to be ignored everywhere.

Reply:
I believe this was already fixed by a later commit. The method is now `flowController.UpdateSendWindow(protocol.ByteCount)` (no return value).